README.md

```diff
@@ -58,7 +65,7 @@ The scripts will:
 1. Complete some post-install configuration - including [machine/node
    linkage
    (#14)](https://github.com/openshift-kni/install-scripts/issues/14),
-   and [configuring a storage VLAN on the `provisioning` interface on
+   and [configuring a storage VLAN on the `Internal` interface on
```

Comment:
can we say "configuring a storage VLAN on the internal interface labeled ???" (I assume it won't be labeled provisioning.